Content.Server/Power/EntitySystems/SubstationSystem.cs

```diff
@@ -121,15 +121,13 @@ public override void Update(float deltaTime)
     private void ConsumeNitrogenBoosterGas(float deltaTime, float scalar, SubstationComponent subs, PowerNetworkBatteryComponent battery, GasMixture mixture)
     {
         var initialN2 = mixture.GetMoles(Gas.Nitrogen);
-        var initialPlasma = mixture.GetMoles(Gas.Plasma);
 
         var molesConsumed = (subs.InitialNitrogenBoosterMoles * battery.CurrentSupply * deltaTime) / (_substationDecayCoeficient * scalar);
 
-        var minimumReaction = Math.Min(initialN2, initialPlasma) * molesConsumed / 2;
+        var minimumReaction = (initialN2);
 
         mixture.AdjustMoles(Gas.Nitrogen, -minimumReaction);
-        mixture.AdjustMoles(Gas.Plasma, -minimumReaction);
-        mixture.AdjustMoles(Gas.NitrousOxide, minimumReaction*2);
+        mixture.AdjustMoles(Gas.NitrousOxide, minimumReaction/2);
```

Comment:
Probably should be just _minimumReaction_, rather than _minimumReaction/2_, since the input is only getting halved, not quartered.